cgsmiles/read_fragments.py

```diff
@@ -239,7 +237,7 @@ def fragment_iter(fragment_str, all_atom=True):
         elif all_atom:
             mol_graph = pysmiles.read_smiles(smile, reinterpret_aromatic=False, strict=False)
             nx.set_node_attributes(mol_graph, bonding_descrpt, 'bonding')
-            nx.set_node_attributes(mol_graph, rs_isomers, 'rs_isomer')
+            nx.set_node_attributes(mol_graph, rs_isomers, 'chiral')
```

Comment:
Or "stereo"?

Reply:
for sure, not `rs_isomer` because that gets smashed up by pysmiles when adding hydrogen. small polling leads at home yield two votes for chiral and one abstention (the dog does not know these things yet).